<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<HTML>
 <HEAD>
   <TITLE> [Mapnik-users] Mapnik 0.7 and SQL errors
   </TITLE>
   <LINK REL="Index" HREF="http://lists.berlios.de/pipermail/mapnik-users/2010-January/index.html" >
   <LINK REL="made" HREF="mailto:mapnik-users%40lists.berlios.de?Subject=Re%3A%20%5BMapnik-users%5D%20Mapnik%200.7%20and%20SQL%20errors&In-Reply-To=%3C1264498837305-4459719.post%40n2.nabble.com%3E">
   <META NAME="robots" CONTENT="index,nofollow">
   <style type="text/css">
       pre {
           white-space: pre-wrap;       /* css-2.1, curent FF, Opera, Safari */
           }
   </style>
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="002867.html">
   <LINK REL="Next"  HREF="002883.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
   <H1>[Mapnik-users] Mapnik 0.7 and SQL errors</H1>
    <B>NopMap</B> 
    <A HREF="mailto:mapnik-users%40lists.berlios.de?Subject=Re%3A%20%5BMapnik-users%5D%20Mapnik%200.7%20and%20SQL%20errors&In-Reply-To=%3C1264498837305-4459719.post%40n2.nabble.com%3E"
       TITLE="[Mapnik-users] Mapnik 0.7 and SQL errors">ekkehart at gmx.de
       </A><BR>
    <I>Tue Jan 26 10:40:37 CET 2010</I>
    <P><UL>
        <LI>Previous message: <A HREF="002867.html">[Mapnik-users] Mapnik 0.7 and SQL errors
</A></li>
        <LI>Next message: <A HREF="002883.html">[Mapnik-users] Mapnik 0.7 and SQL errors
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#2877">[ date ]</a>
              <a href="thread.html#2877">[ thread ]</a>
              <a href="subject.html#2877">[ subject ]</a>
              <a href="author.html#2877">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>

Dane Springmeyer wrote:
&gt;<i> 
</I>&gt;&gt;<i> but I wondered about the error handling. When running mapnik from  
</I>&gt;&gt;<i> the python
</I>&gt;&gt;<i> binding (generate_image.py), I do not see any errors,
</I>&gt;<i> 
</I>&gt;<i> After fixing the above missing column, or before? If after this may be  
</I>&gt;<i> Mapnik only throwing an error due to renderd not being able to  
</I>&gt;<i> connect, or something different than a query syntax error...
</I>&gt;<i> 
</I>
Before, I haven't touched anything yet, of course. :-)

Shouldn't such an exception be visible in python?


Dane Springmeyer wrote:
&gt;<i> 
</I>&gt;&gt;<i> when running mapnik
</I>&gt;&gt;<i> from C++ (rendered), there is a segmentation fault.
</I>&gt;<i> 
</I>&gt;<i> Where, how, and when?  renderd likely needs to be improved to catch  
</I>&gt;<i> the `mapnik::datasource_exception`, but it is hard to know without  
</I>&gt;<i> more info.
</I>&gt;<i> 
</I>
Hard to tell. Sven, the devserver admin is looking into this, I'm trying to
understand the intended behaviour in order to help him.

bye
          Nop

-- 
View this message in context: <A HREF="http://n2.nabble.com/Mapnik-0-7-and-SQL-errors-tp4455570p4459719.html">http://n2.nabble.com/Mapnik-0-7-and-SQL-errors-tp4455570p4459719.html</A>
Sent from the Mapnik Users mailing list archive at Nabble.com.

</PRE>

<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="002867.html">[Mapnik-users] Mapnik 0.7 and SQL errors
</A></li>
	<LI>Next message: <A HREF="002883.html">[Mapnik-users] Mapnik 0.7 and SQL errors
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#2877">[ date ]</a>
              <a href="thread.html#2877">[ thread ]</a>
              <a href="subject.html#2877">[ subject ]</a>
              <a href="author.html#2877">[ author ]</a>
         </LI>
       </UL>

<hr>
<a href="https://lists.berlios.de/mailman/listinfo/mapnik-users">More information about the Mapnik-users
mailing list</a><br>
</body></html>
